app/src/components/inputs/inputSlider/inputSlider.jsx

```diff
@@ -0,0 +1,69 @@
+import Slider from 'rc-slider';
+import React, { Component } from 'react';
+import 'rc-slider/assets/index.css';
+import 'rc-tooltip/assets/bootstrap.css';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './inputSlider.scss';
+
+const cx = classNames.bind(styles);
+
+export class InputSlider extends Component {
+  static propTypes = {
+    options: PropTypes.array,
+    value: PropTypes.string,
+    onChange: PropTypes.func,
+  };
+  static defaultProps = {
+    options: [],
+    value: '',
+    onChange: () => {},
+  };
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
There's no need to store anything in state in this case.
Caching options converted to object might be a good idea, but in this case there should be a way to sync this with props which will make this component more complicated. I don't think it's necessary at this moment.